docs/cristin.ttl

```diff
@@ -0,0 +1,57 @@
+@prefix rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs:   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd:    <http://www.w3.org/2001/XMLSchema#> .
+@prefix cr:     <http://api.cristin.no/> .
+
+#Info about rdfs types
+#https://www.w3.org/TR/rdf-schema/#ch_resource
+
+cr:hasExactlyOne
+    rdf:type    rdf:Property ;
+    rdfs:label  "property with exactly one value" ;
+    rdfs:range  rdfs:Resource ;
+    rdfs:domain cr:Result .
+
+cr:Result
+    rdf:type         rdfs:Class ;
+    cr:hasExactlyOne cr:Id ;
+    cr:hasExactlyOne cr:arstall ;
+    cr:hasExactlyOne cr:dato_opprettet ;
+    cr:hasExactlyOne cr:dato_opprettet ;
+    cr:hasExactlyOne cr:varbeidhovedkatkode ;
+    cr:hasExactlyOne cr:varbeidunderkatkode .
+
+cr:Id
+    rdf:type    rdf:Property ;
+    rdfs:label  "Identifier of Cristin Result entry" ;
+    rdfs:range  xsd:positiveInteger ;
```

Comment:
For reading simplicity, reorder the range and domain so that domain precedes range. It makes no sense outside this document, of course, but for a human reader, it makes it easier.